statusHandler/statusMetricsProvider.go

```diff
@@ -209,6 +211,27 @@ func (sm *statusMetrics) EnableEpochsMetrics() map[string]interface{} {
 	enableEpochsMetrics[common.MetricDelegationManagerEnableEpoch] = sm.loadUint64Metric(common.MetricDelegationManagerEnableEpoch)
 	enableEpochsMetrics[common.MetricDelegationSmartContractEnableEpoch] = sm.loadUint64Metric(common.MetricDelegationSmartContractEnableEpoch)
 	enableEpochsMetrics[common.MetricIncrementSCRNonceInMultiTransferEnableEpoch] = sm.loadUint64Metric(common.MetricIncrementSCRNonceInMultiTransferEnableEpoch)
+	enableEpochsMetrics[common.MetricBalanceWaitingListsEnableEpoch] = sm.loadUint64Metric(common.MetricBalanceWaitingListsEnableEpoch)
+	enableEpochsMetrics[common.MetricWaitingListFixEnableEpoch] = sm.loadUint64Metric(common.MetricWaitingListFixEnableEpoch)
+
+	numNodesChangeConfig := sm.loadUint64Metric(common.MetricMaxNodesChangeEnableEpoch + "_count")
+
+	nodesChangeConfig := make([]map[string]interface{}, 0)
+	for i := uint64(0); i < numNodesChangeConfig; i++ {
+		maxNodesChangeConfig := make(map[string]interface{})
+
+		epochEnable := fmt.Sprintf("%s%d%s", common.MetricMaxNodesChangeEnableEpoch, i, common.EpochEnableSuffix)
+		maxNodesChangeConfig[common.EpochEnableSuffix] = sm.loadUint64Metric(epochEnable)
```

Comment:
I don't think it's ok to export the key as `_enable_epoch`. Might rename it to be consistent with the other metrics (`erd_..`) but you can still use this variable in order to compose the keys

Reply:
right :+1: changed the key to be consistent with the other metrics; applied the changes for ratings metrics also